### PR TITLE
bot activated

### PR DESCRIPTION
**Pull request by aishux**
## Related Issue 

Closes: #119 

### Describe the changes you've made

Added a script that can auto-label issues with the GSSOC21 label whenever contributors ask to work under that label.

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Enhancement in the existing features
- [ ] Other

## Mention any unusual behaviour of your code (Write NA if not)
None

## Checklist:

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly whereever it was hard to understand.
- [x] My changes generate no new warnings.
- [x] I have verified/tested my code by running it locally.

## Additional Info (optional)
Please merge my request and assign me a GSSOC tag and a level tag.

### CODE REVIEW

File: .github/workflows/label_issue.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+    types: ['opened']
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: Renato66/auto-label@v2
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          ignore-comments: true
+          labels-synonyms: '{"GSSOC21":["GSSOC","GSSOC21","gssoc","Gssoc","GSSoC","gssoc21"]}'
```
Comment (smaranjitghose) [inline .github/workflows/label_issue.yml]: Just use gssoc21 please

**Review by smaranjitghose — APPROVED** [review]

File: .github/workflows/label_issue.yml (hunk 1 above)
Comment (aishux) [reply .github/workflows/label_issue.yml]: Updated

**Comment by smaranjitghose** [comment]
Nice work @aishux

**Comment by aishux** [comment]
> Nice work @aishux
> 

Thankyou :blush: Please give me a level tag.

**Pull request merged**